### DISCUSSION TITLE
Archiving/unarchiving: additional formats

### DISCUSSION THREAD

**Discussion by JayBrown** [mikekazakov/nimble-commander#229] [Ideas]
I can't check which is/are already implemented in NC, because the software keeps crashing on me, but I assume that zip is part of the deal. Would also be great to have built-in support for tar, gz, bz2, xz (and combinations of the former like txz), 7z, and possibly rar (unarchiving only). Moreover, there could be an additional functionality when holding the ALT key, while selecting the create archive menu item, e.g. encrypt with password.

**Comment by m-cerveny** [comment]
I'm very much missing a dialog like when doing Copy/Move operations. Also, possibility to select compression method is missing. What is there is perfectly functional, but it's rather basic.

**Comment by BashTux1** [comment]
+1 for ability to select different compression formats.
